Bang/Game/Gameplay.cs

```diff
@@ -150,5 +150,61 @@ public void ChooseCard(BangGameCard card, Player player)
         {
             state = state.ApplyReplyAction(player, card);
         }
+
+        public void StartNextPlayerTurn()
```

Comment:
Method `StartNextPlayerTurn()` looks too long. I suppose to inroduce two methods: first one will handle dynamite card logic, second one will handle jail card logic.

Reply:
done